l10n/en/firefox/browsers/chromebook.ftl

```diff
@@ -0,0 +1,29 @@
+# This Source Code Form is subject to the terms of the { -brand-name-mozilla } Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/browsers/chromebook/
+
+# HTML page title
+
+get-firefox-browser = Get { -brand-name-firefox-browser } for { -brand-name-chromebook }
```

Comment:
Fluent IDs should use a page specific naming prefix: https://bedrock.readthedocs.io/en/latest/l10n.html#ftl-string-ids

Reply:
adding `browsers-chromebook-` at the beginning of all IDs